lib/events/dynamoevents/dynamoevents_test.go

```diff
@@ -81,6 +84,10 @@ func (s *DynamoeventsSuite) SetUpTest(c *check.C) {
 	c.Assert(err, check.IsNil)
 }
 
+func (s *DynamoeventsSuite) TestPagination(c *check.C) {
+	s.EventPagination(c)
```

Comment:
Would it be too cumbersome to introduce this as a test based on the `testing` module in accordance with our new testing guidelines?

Reply:
I think it is, in order to not introduce a lot of code duplication and weird logic it only really makes sense if we rewrite all of these suite tests which I'd rather not do in this already gigantic (in terms of delta) PR.